Changelogs for Digital Twin Manager UI and Microservice 1020.3.1 (#2692)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020-1-20-title-case-for-app-introduction-has-been-changed-to-digital-twin-manager.md

---
date: ""

> title: "Title case for app introduction has been changed to Digital Twin Manager"

Reviewer: ```suggestion
title: "Digital Twin Manager now uses title case, consistent with all other application titles"
```

> title: "Digital Twin Manager now uses title case, consistent with all other application titles"

Reviewer 2: ```suggestion
title: "Digital Twin Manager application titles now use title case"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1548"
version: "1020.1.20"
---

> Previously the application name for DTM is 'Digital twin manager'. Now it has been changed to 'Digital Twin Manager'.

Reviewer: Copying the changelog which DM team put for the change : https://cumulocity.com/docs/change-logs/?productarea=.productarea-device-management-connectivity#ui-c8y-1020-32-0-changed-dm-title-to-be-title-cased

```suggestion
To improve readability and consistency, the titles of Digital twin manager application now use title case (Digital Twin Manager) instead of sentence case (Digital twin manager). The functionality of the application remains the same, only the casing of the titles has been modified.'.
```

> To improve readability and consistency, the titles of Digital twin manager application now use title case (Digital Twin Manager) instead of sentence case (Digital twin manager). The functionality of the application remains the same, only the casing of the titles has been modified.'.

Reviewer 2: ```suggestion
To improve readability and consistency with other applications, the titles of the Digital Twin Manager application now use title case (Digital Twin Manager) instead of sentence case (Digital twin manager). The functionality of the application remains the same, only the casing of the titles has been modified.
```
